nlb-target-group.tf

```diff
@@ -10,7 +10,7 @@ resource "aws_lb_listener" "ecs_tcp" {
 }
 
 resource "aws_lb_target_group" "ecs_default_tcp" {
-  name        = "ecs-${var.cluster_name}-${var.name}-tcp"
+  name        = "ecs-${var.name}-tcp"
```

Comment:
I think it's better to keep the cluster_name and name here, otherwise if the same app name exists in 2 cluster, we will have a name conflict.

You can use something like this to limit the name to 32 chars, by limiting the cluster_name-name to 24 chars and adding a 4 random chars at the end: https://github.com/DNXLabs/terraform-aws-ecs-app/blob/master/alb-target-group.tf#L153

```
resource "random_string" "alb_prefix" {
  length  = 4
  upper   = false
  special = false
}

resource "aws_lb_target_group" "ecs_default_tcp" {
name        =  format("%s-%s-tcp", substr("${var.cluster_name}-${var.name}", 0, 24), random_string.alb_prefix.result
}
```